exercises/triangle/triangle_test.rb

```diff
@@ -3,91 +3,130 @@
 require 'minitest/autorun'
 require_relative 'triangle'
 
+# Test data version:
+# b806246
 class TriangleTest < Minitest::Test
-  def test_equilateral_triangles_have_equal_sides
-    assert_equal :equilateral, Triangle.new(2, 2, 2).kind
+  def test_triangle_is_equilateral_if_all_sides_are_equal
+    triangle = Triangle.new([2, 2, 2])
+    assert triangle.equilateral?, "Expected 'true', triangle is equilateral."
   end
 
-  def test_larger_equilateral_triangles_also_have_equal_sides
+  def test_triangle_is_not_equilateral_if_any_side_is_unequal
     skip
-    assert_equal :equilateral, Triangle.new(10, 10, 10).kind
+    triangle = Triangle.new([2, 3, 2])
+    refute triangle.equilateral?, "Expected 'false', triangle is not equilateral."
   end
 
-  def test_isosceles_triangles_have_last_two_sides_equal
+  def test_triangle_is_not_equilateral_if_no_sides_are_equal
     skip
-    assert_equal :isosceles, Triangle.new(3, 4, 4).kind
+    triangle = Triangle.new([5, 4, 6])
+    refute triangle.equilateral?, "Expected 'false', triangle is not equilateral."
   end
 
-  def test_isosceles_triangles_have_first_and_last_sides_equal
+  def test_all_zero_sides_are_illegal__so_the_triangle_is_not_equilateral
```

Comment:
Oh yeah, decided to replace commas with underscores too, but I see is not that really necessary. xD  Ty!